python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -102,7 +105,7 @@ def fetchReferences(self, dataRef, exposure):
     def _makeArgumentParser(cls):
         parser = lsst.pipe.base.ArgumentParser(name=cls._DefaultName)
         parser.add_id_argument("--id", "deepCoadd_forced_src", help="data ID, with raw CCD keys + tract",
-                               ContainerClass=lsst.coadd.utils.CoaddDataIdContainer)
+                               ContainerClass=lsst.coadd.utils.ExistingCoaddDataIdContainer)
```

Comment:
Indeed I did go with the global interpretation, so the merge pipe_tasks has been reverted.  I did merge the addition of the class to coadd_utils, so if/when we every want to use it, it's there.